BlueCheese/Hubs/LobbyHub.cs

```diff
@@ -18,12 +18,12 @@ public LobbyHub(IGameManager gameManager, ILogger<LobbyHub> logger)
             _logger = logger;
         }
 
-        public async Task ClientStartedNewGame(NewGameStarted newGame)
+        public async Task<IGame> ClientStartedNewGame(NewGameStarted newGame)
         {
             if(newGame==null) throw new ArgumentNullException(nameof(newGame));
             _logger.LogInformation("LobbyHub.ClientStartedNewGame {@newGame}", newGame);
 
-            await _gameManager.StartNewGameAsync(Context.ConnectionId, newGame).ConfigureAwait(false);
+            return await _gameManager.StartNewGameAsync(Context.ConnectionId, newGame).ConfigureAwait(false);
```

Comment:
the docs show these client initiated methods as returning nothing... separate signals sent when things happen....

does this actually return the game object? and if so - can you update the lobby.js in this project to print it out to console or something like that?

Reply:
yeah, i'll add that in

Reply:
what docs you referring to?

Reply:
Game object is used for the front end to be able to go to the game using the game GUID.